Dragonflight/ShamanElemental.lua

```diff
@@ -248,14 +272,20 @@ spec:RegisterAuras( {
         type = "Magic",
         max_stack = 1,
     },
+	heat_wave = { --hidden aura, doesn't track correctly
```

Comment:
If this is fully hidden, we need to generate a fake aura based on Primordial Wave's last cast time, and queue procs of Lava Surge around the fake aura.